readme.md

```diff
@@ -86,9 +87,9 @@ DATADIR = "pocketsphinx/test/data"
 
 # Create a decoder with certain model
 config = Decoder.default_config()
-config.set_string('-hmm', path.join(MODELDIR, 'en-us/en-us'))
```

Comment:
My pocketsphinx  version is 0.1.3 on Windows-10 32bit.
There is only one en-us folder